### PR TITLE
Basic template

### PR DESCRIPTION
**Pull request by mbrousil**
Basic bookdown template including references. Sections named for parts of a paper.

### CODE REVIEW

File: index.Rmd [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+# Intro
+
+This is a _sample_ book written in **Markdown**. 
+
+## Render book
+
+You can render the HTML version of this example book without changing anything:
+
+1. Find the **Build** pane in the RStudio IDE, and
```
Comment (steeleb) [inline index.Rmd]: Maybe add a screen shot?

Comment (steeleb) [inline index.Rmd]: Maybe add a screen shot?

File: index.Rmd [hunk 2]
```diff
@@ -0,0 +1,34 @@
+--- 
+title: "Bookdown Template"
+author:
+date: "`r Sys.Date()`"
+site: bookdown::bookdown_site
+documentclass: book
+bibliography: [book.bib, packages.bib]
+# url: your book url like https://bookdown.org/yihui/bookdown
```
Comment (steeleb) [inline index.Rmd]: suggestion:
# url: your book url like https://bookdown.org/USERNAME/REPO

File: _bookdown.yml
Comment (steeleb) [inline _bookdown.yml:1]: does this automatically populate? Maybe note in the readme the minimum things someone needs to edit in order to make a bookdown?

File: README.md [hunk 3]
```diff
@@ -1 +1,13 @@
...
+
+This is a minimal example of a book based on R Markdown and **bookdown** (https://github.com/rstudio/bookdown). 
+
+This template provides a skeleton file structure that you can edit to create your book. 
+
+The contents inside the .Rmd files provide some pointers to help you get started, but feel free to also delete the content in each file and start fresh.
+
+Additional resources:
+
+The **bookdown** book: https://bookdown.org/yihui/bookdown/
```
Comment (steeleb) [inline README.md:12]: 'We highly suggest you review chapter 1 of this book before forking this repo!'

**Review by steeleb — CHANGES_REQUESTED** [review]
Dig it!

Can you add a bit more high-level instruction for this repo - as is in: this fake book has 5 chapters (_list the chapters_) that are all pulled together in the `index.Rmd` file. 

Related: should `01-abstract.Rmd` be a thing?

File: index.Rmd (hunk 1 above)
Comment (mbrousil) [reply index.Rmd]: This was auto-populated. I've never used this myself so I deleted it and now just show the code option.

**Comment by steeleb** [comment]
Suhweet. 

This looks great! Merging this sucker.

**Pull request merged**